ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx

```diff
@@ -37,6 +35,32 @@ export const PodsLogsViewer = (props: PodLogsProps & {fullscreen?: boolean}) =>
     const page = props.page;
     const setPage = props.setPage;
     const [viewPodNames, setViewPodNames] = useState(false);
+    const [logLinesField, setLogLinesField] = useState(null);
```

Comment:
I think we are planning to replace `DataLoader` later with a hook-based analog. Can we keep using DataLoader until hook based version is available ?

Reply:
Good idea. The `useData` hook is available in the `argo-ux` package now, so I can either switch to that, or use `DataLoader` for now